auth-server/src/model/EmailUndoToken.ts

```diff
@@ -0,0 +1,45 @@
+import { Model } from 'objection';
+import connection from './connection';
+
+Model.knex(connection);
+
+// token to undo email change
+export default class EmailUndoToken extends Model {
```

Comment:
I know I'm annoying with names but if we had UndoEmail**Change**Token as a name, we wouldn't need a comment. "UndoEmail" is used everywhere in this PR, it'd be great to have the "Change" added to the names of functions/files/table to make it clear that only the change is undone, and not the email.

Reply:
done